### ISSUE TITLE
Aligned features do not contain tolerances

### ISSUE THREAD

**Issue by xtrojak** [RECETOX/recetox-aplcms-archive#176]
As a relic from the previous version, the output of `align_features` function contained _unchanged_ (unless they were `NA` on input which isn't possible at that step) `rt_tol_relative` and `mz_tol_relative` values. These are then used in subsequent steps in both unsupervised and hybrid workflows, but the function `create_aligned_feature_table` (used instead `align_features`) does not return them anymore.

https://github.com/RECETOX/recetox-aplcms/blob/aa11247232ead7a2a2190ec13eb00c349cbc6960/R/unsupervised.R#L221

https://github.com/RECETOX/recetox-aplcms/blob/aa11247232ead7a2a2190ec13eb00c349cbc6960/R/unsupervised.R#L272

TL;DR, we have to use these values from `adjusted_clusters` instead.

**Comment by hechth** [comment]
@xtrojak I think this should actually be fixed now? Or do we want to return them again from the function?

**Comment by xtrojak** [comment]
@hechth no, it's not fixed.
